test/Test/nlp.jl

```diff
@@ -16,4 +16,5 @@ const MOI = MathOptInterface
         )
     )
     MOI.Test.test_linear_mixed_complementarity(mock, config)
+    MOI.Test.test_qp_mixed_complementarity(mock, config)
```

Comment:
You need to set the solution in the mock optimizer like I do here:
https://github.com/JuliaOpt/MathOptInterface.jl/pull/913/files#diff-00b88401713e1c20041db74f14c5ccd1R12-R17